src/schema/setup.rs

```diff
@@ -0,0 +1,39 @@
+use pgx::*;
+
+// First we need to define some objects which should be defined by `promscale`
+// This is mostly so that the extension can be installed standalone, which is
+// convenient for development.
+extension_sql!(
+    r#"
+CREATE OR REPLACE FUNCTION swallow_error(query text) RETURNS VOID AS
```

Comment:
This should specify a schema we own right? Just like we do further down below. Otherwise it might potentially replace, then drop, a function of the same name. I haven't checked to see if we're implicitly creating this in a schema other than `public`, but it caught my eye anyway.

Reply:
👍